src/styles/supportModal.css

```diff
@@ -2,6 +2,10 @@
   min-height: 8em;
 }
 
+[role=banner] .coz-support-modal {
+  max-width: inherit;
```

Comment:
inherit ? Why not none here ?

Reply:
Also I do not understand why everywhere in bar you use [role=banner] and not something like "cozy-bar".

Reply:
You're right, none is a better value 👍 
For the [role="banner"] it's an aria value for the role attribute (https://www.w3.org/TR/wai-aria/#landmark_roles)

Reply:
Yeah I know it's aria stuff but it still feels weird to me to do this instead of `.cozy-bar`.

Reply:
Since we also use [role=application] for the application part, it's consistent for me.